.drone.starlark

```diff
@@ -0,0 +1,1073 @@
+config = {
+	'app': 'files_antivirus',
+	'rocketchat': {
+		'channel': 'builds',
+		'from_secret': 'private_rocketchat'
+	},
+
+	'branches': [
+		'master'
+	],
+
+	'appInstallCommand': 'make',
+
+	'codestyle': {
+		'ordinary' : {
+			'phpVersions': [
+				'7.0',
+				'7.1',
+				'7.2',
+				'7.3',
+			],
+		},
+	},
+
+	'phpunit': {
+		'allDatabases' : {
+			'phpVersions': [
+				'7.0',
+			]
+		},
+		'reducedDatabases' : {
+			'phpVersions': [
+				'7.1',
+				'7.2',
+				'7.3',
+			],
+			'databases': [
+				'sqlite',
+				'mariadb:10.2',
+			],
+			'coverage': False
+		},
+	},
+
+	'acceptance': {
+		'webUI': {
+			'suites': {
+				'webUIActivityList': 'webUIActList',
+			},
+			'browsers': [
+				'chrome',
+				'firefox'
+			],
+			'extraApps': [
+				'activity'
+			],
+		},
+		'api': {
+			'suites': [
+				'apiAntivirus'
+			],
+			'databases': [
+				'mariadb:10.2',
+				'mysql:5.5',
+				'mysql:5.7',
+				'postgres:9.4',
+				'oracle'
+			],
+		},
+		'api-objectstore': {
+			'suites': {
+				'apiAntivirus': 'apiAntivirusObjectstore',
+			},
+			'databases': [
+				'mariadb:10.2',
+			],
+			'extraApps': {
+				'files_primary_s3': 'composer install'
+			},
+			'extraSetup': {
+				'name': 'configure-app',
+				'image': 'owncloudci/php:7.0',
+				'pull': 'always',
+				'commands': [
+					'cd /var/www/owncloud/server/apps/files_primary_s3',
+					'cp tests/drone/scality.config.php /var/www/owncloud/server/config',
+					'cd /var/www/owncloud/server',
+					'php occ s3:create-bucket owncloud --accept-warning',
+					'php occ config:app:set --value "clamav" files_antivirus av_host',
+					'php occ config:app:set --value "daemon" files_antivirus av_mode',
+					'php occ config:app:set --value "3310" files_antivirus av_port',
+				]
+			},
+			'extraServices': [
+				{
+					'name': 'clamav',
+					'image': 'dinkel/clamavd',
+					'pull': 'always',
+				},
+				{
+					'name': 'scality',
+					'image': 'owncloudci/scality-s3server',
+					'pull': 'always',
+					'environment': {
+						'HOST_NAME': 'scality'
+					}
+				}
+			]
+		},
+	},
+
+	'defaults': {
+		'phpunit': {
+			'extraServices': [
+				{
+					'name': 'dummy-clamav',
+					'image': 'owncloudci/php:7.0',
+					'pull': 'always',
+					'commands': [
+						# dummy clamav required for unit tests
+						'php tests/util/avirserver.php',
+					]
+				}
+			],
+			'extraEnvironment': {
+				'AVIR_HOST': 'dummy-clamav'
```

Comment:
The "unit" tests do not mock everything. They depend on having an antivirus service.
The unit test code has been modified so, instead of being hard-coded to `0.0.0.0` or `127.0.0.1`, it will connect with `AVIR_HOST` if that environment variable is defined.